tests/benchdnn/conv/deconv.cpp

```diff
@@ -238,8 +238,10 @@ void check_known_skipped_case(const prb_t *prb, res_t *res) {
         const bool bwd_tag_ok
                 = !((prb->dir == BWD_W || prb->dir == BWD_WB) && stag_is_axb);
         const bool tag_ok = fwd_tag_ok && bwd_tag_ok;
+        const bool unimplemented_ok = (res->state == UNIMPLEMENTED);
```

Comment:
We should limit the set of unimplemented cases that we consider valid. We can't say that all unimplemented cases are valid.
```cpp
// Deconvolution doesn't support s8/u8 data type.
const bool data_type_ok = prb.cfg[SRC].dt != dnnl_s8 && prb.cfg[SRC].dt != dnnl_u8;
if (!pad_ok || !out_ok || !post_ops_ok || !tag_ok || !data_type_ok) {
    // ...
}
```

Reply:
Okay, I have changed the check

Reply:
Thanks